igor-web/src/main/java/com/netflix/spinnaker/igor/gcb/GoogleCloudBuildAccount.java

```diff
@@ -71,6 +74,22 @@ public Build getBuild(String buildId) {
     return googleCloudBuildParser.parse(buildString, Build.class);
   }
 
+  public List<BuildTrigger> listTriggers() {
```

Comment:
Would like to see a simple test in `GoogleCloudBuildAccountSpec` for `listTriggers` and `runTrigger`...really just ensuring that we pass the data along to the `client` and return the response.  The first test (`createBuild creates a build and returns the result`) should be a good example.

Reply:
I missed that one. Didn't search for references in groovy classes. Will add some tests. Thanks!